### DISCUSSION TITLE
Rich table output is truncated for result=ansi

### DISCUSSION THREAD

**Discussion by NikosAlexandris** [pawamoy/markdown-exec#33] [Q&A]
Rich table output in the console is truncated (i.e. table column names are incomplete), for example : https://nikosalexandris.github.io/rekx/how_to/inspect/. See in [this pages source code](https://github.com/NikosAlexandris/rekx/raw/main/docs/how_to/inspect.md) the output of  commands wrapped inside a block

``` markdown
    ``` bash exec="true" result="ansi" source="above"
    ```
```

Any idea how to "fix" this ?
See also : https://github.com/squidfunk/mkdocs-material/issues/4964#issuecomment-1898039123.

**Comment by pawamoy** [comment]
Thanks for the report. To fix this I suppose we have to specify a larger terminal width. I'll see how we can do that.
